requirements/default.txt

```diff
@@ -67,8 +67,8 @@ https://github.com/django-security/django-axes/archive/778f208cc1d3cc42ecb756f73
 https://github.com/mozilla/django-badger/archive/3e9fded1a5bd17a69a852d410eaa4b79686bdd32.tar.gz#egg=django-badger==0.0.0.1
 
 # django-cache-machine: master~1
```

Comment:
Is this still "master tip minus one commit"? If not, then I'd nix the `~1` part on this line.